docs/data/material/customization/theming/theming.md

```diff
@@ -45,7 +45,7 @@ const theme = createTheme({
 ```
 
 :::warning
-⚠️ `vars` is a private field for [CSS theme variables](material-ui/experimental-api/css-theme-variables/overview/). It will throw an error if you try to pass a value to it:
+`vars` is a private field for [CSS theme variables](/material-ui/experimental-api/css-theme-variables/overview/). It will throw an error if you try to pass a value to it:
```

Comment:
1. I have removed ⚠️ so we can normalize how warning are displayed in the docs
2. I think that we should stop using the `/overview/` pattern, `/` should be enough for this

- https://mui.com/material-ui/getting-started/overview/ -> https://mui.com/material-ui/getting-started/
- https://mui.com/material-ui/experimental-api/css-theme-variables/overview/ -> https://mui.com/material-ui/experimental-api/css-theme-variables/
- etc.